pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -219,27 +220,25 @@ func (c *Reconciler) reconcile(ctx context.Context, service *v1alpha1.Service) e
 	ss := &service.Status
 	ss.PropagateRouteStatus(&route.Status)
 
-	// Apply additional logic, once the generic data has been propagated.
-	switch {
-	case service.Spec.RunLatest != nil:
-		runLatestCheckTrafficMigrated(ss, &route.Status, &config.Status)
+	if service.Spec.RunLatest != nil || service.Spec.Release != nil {
```

Comment:
Well there's still pinned, it's deprecated, but supported.

Reply:
Yeah, but it's a nop for Pinned.  I think I have a mild preference for `Manual != nil` as well because if we were to add a new mode, this would either do the right thing or the author would have to adjust this logic for the Service to become Ready.

Reply:
Doesn't matter at all, since we don't reconcile `manual` anyway.

Reply:
So we don't reconcile if we're in manual, so I removed the check altogether.